test/cli.js

```diff
@@ -2,24 +2,40 @@
 // Licensed under the Apache License, Version 2.0. See LICENSE
 'use strict';
 
+const path = require('path');
 const childProcess = require('child_process');
 const expect = require('chai').expect;
 
 describe('CLI', function() {
   describe('url', () => {
     it('should throw error if a url is not provided by cli', () => {
       try {
-        childProcess.execSync('node bin/cli.js --config');
+        childProcess.execSync('bin/cli.js');
       } catch (e) {
         expect(e.message).to.contain('No url entered..');
       }
     });
+
     it('should throw error if a url is not provided either by config or by cli', () => {
       try {
-        childProcess.execSync('node bin/cli.js --config=./test/fixtures/empty-config.js');
+        childProcess.execSync('bin/cli.js --config=./test/fixtures/empty-config.js');
       } catch (e) {
         expect(e.message).to.contain('No url entered..');
       }
     });
+
+    it('should not throw any error if a url is provided from cli', () => {
+      return childProcess.exec('bin/cli.js https://example.com', error => {
```

Comment:
Sam is working on https://github.com/GoogleChrome/lighthouse/issues/2092, so it should fix our problem in future. But for now, we can make some ugly fix.